composer.json

```diff
@@ -1,13 +1,13 @@
 {
-    "name": "doge-dev/laravel-sms-verification",
-    "description": "Library for sending out and verifying SMS codes using AWS SNS",
+    "name": "freelabois/laravel-sms-verification",
```

Comment:
can not change this

Reply:
WOW, my bad, I was in a tight deadline on that commit, will revert this change and remove my repo if merged